docs/_data/fuzzers.yaml

```diff
@@ -18,8 +18,8 @@ Fuzzers:
   year: 2020
   url: https://arxiv.org/pdf/1911.00621.pdf
   dblp: journals/corr/abs-1911-00621
-  repo:
-  dir:
+  repo: https://github.com/andreafioraldi/weizz-fuzzer
+  dir: weizz_qemu
```

Comment:
It's odd to me this directory is weizz_qemu and not weizz.

Reply:
Since it is a binary only fuzzer i think - https://github.com/andreafioraldi/weizz-fuzzer , needs qemu. also @vanhauser-thc did it explicitly in https://github.com/google/fuzzbench/pull/681/files

Reply:
the reason is that weizz can work with llvm and with qemu, and this is the qemu version. so the naming is correct. (the llvm has less features as far as I understood andrea)